recognition/advanced_tree_search.py

```diff
@@ -448,6 +446,9 @@ def create_config(
         post_config.flf_lattice_tool.network.sink.warn_on_empty_lattice = True
         post_config.flf_lattice_tool.network.sink.error_on_empty_lattice = False
 
+        post_config["*"].intra_op_parallelism_threads = kwargs["cpu"]
```

Comment:
> Can we figure out what is the exact position in the config and use that instead of "*"?

I was too lazy for that....

> Why not make kwargs["cpu"] an explicit argument to the function?

Sure